lesson2/README.md

```diff
@@ -1,14 +1,85 @@
 # Lesson 2 - Android Basics
-
+### Your first Android application!
 Today in class we'll walk you through making your first Android application. Found in the lesson2 folder is a copy of the application that we will create in class.
 
-## [Activities](https://developer.android.com/guide/components/activities.html) and [Fragments](https://developer.android.com/guide/components/fragments.html)
+## Create a Blank App
+To begin with, we'll create a default app and walk you through the different components.
 
-more info
+1. Create a new project in Android Studio
+2. Name it something akin to "Lesson2HW" and save it in your forked repo
+3. Click "Basic Activity"
+4. Check "Use a Fragment"
 
+Great, you just created your very first app!
 
-## Extra Resources
+## Explore Around
+Android Studio just created a bunch of files and boilerplate code to get you started. Before we write any code, let's explore the different files that were just created.
+
+There are generally three different types of files that you'll need to know about in Android Studio:
+
+1. **Java files** are where your backend logic goes.
+2. **XML Layout files** are where you define the frontend layout of your screens.
+3. **Configuration files** are used by Android Studio to configure your app.
+
+You'll be writing most of your code in Java files.
+### 1. Java Files
+When you interact with an app on your phone, there are different "screens" that you can go to. Snapchat, for example has a "friends list" screen, a "picture-taking" screen, a "settings" screen, and so on.  The settings screen displays different things and needs completely different logic than the picture-taking screen.  Android separates these "screens" into different files called **activities** and **fragments**.
+
+#### [Activities](https://developer.android.com/guide/components/activities.html) (ex. MainActivity.java)
+
+Activities contain one or many fragments inside of them.  It handles the passing of information between fragments.  Most simple apps (like the ones you'll be making in the beginning of this class) will only need one activity.
+
+#### [Fragments](https://developer.android.com/guide/components/fragments.html) (ex. MainActivityFragment.java)
+Fragments are basically what we described as "screens" in our Snapchat example. Each fragment does a separate task and shows different components to the user. If you were making Snapchat, you might have a fragment for each swipeable screen. Fragments are useful for:
```

Comment:
I remember how we wanted people to be aware of latest stuff happening in Android. This is a pretty nice talk that happened at Google I/O just a few weeks ago. And I think it will be a good exposure for them.
https://www.youtube.com/watch?v=k3IT-IJ0J98